priv/repo/migrations/20250830152040_create_demographic_profile_table.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Dbservice.Repo.Migrations.CreateDemographicProfileTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:demographic_profile) do
+      add :category_id, :integer
+      add :gender, :string
+      add :caste, :string
+      add :pwd, :string
+      add :income, :string
+      add :religion, :string
+      add :defence_ward, :string
+      add :nationality, :string
+      add :ews_ward, :string
+      add :language, :string
+      add :urban_rural, :string
+      add :region, :string
+
+      timestamps()
+    end
+
+    create index(:demographic_profile, [:category_id])
+    create index(:demographic_profile, [:gender])
+    create index(:demographic_profile, [:caste])
```

Comment:
Do we expect queries to filter on these fields individually, or mostly in combination (e.g. WHERE category_id = ? AND gender = ? AND caste = ?)?